modules/nf-commons/src/main/nextflow/util/CmdLineOptionMap.groovy

```diff
@@ -39,7 +39,7 @@ class CmdLineOptionMap implements CacheFunnel {
     }
 
     def getFirstValue(String key) {
-        getFirstValueOrDefault(key, '')
+        getFirstValueOrDefault(key, null)
```

Comment:
Usually, it's used `null` for absences of value

Reply:
Got it. Thanks.